src/Elmish.WPF/Elmish.WPF.fsproj

```diff
@@ -54,12 +54,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Elmish" Version="[3.1.0, 3.99]" />
+    <PackageReference Include="Elmish" Version="[4.0.0, 4.99]" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="[6.0.1, 99]" />
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Update="FSharp.Core" Version="[6.0.5, 99]" />
+    <PackageReference Update="FSharp.Core" Version="6.0.7" />
```

Comment:
I don't see any reason to change this line.  Elmish v4 is using version `4.7.*`.

https://github.com/elmish/elmish/blob/8b812950f98b7784406b87cb63a1219bb4c5b795/netstandard/Elmish.fsproj#L16

Reply:
Interesting. I went by the final calculated dependency listed on NuGet, which is ">= 6.0.7"
https://www.nuget.org/packages/Elmish#dependencies-body-tab

Reply:
That is strange.

What does the package manager in Visual Studio say?  Can you share a screenshot?

Reply:
From VS 2022 package manager:
![image](https://user-images.githubusercontent.com/1030435/215346832-666427d0-78af-45c8-b5be-7d200487ac39.png)

Reply:
I see the problem:
The url you sent was from a different branch:
![image](https://user-images.githubusercontent.com/1030435/215347384-400ef53e-7e23-43ae-ba4f-b3b83e68d2e3.png)


Here is the v4 branch:
![image](https://user-images.githubusercontent.com/1030435/215347409-8c981837-54cc-43f6-a713-e1fa4bae012e.png)

Reply:
I notice that it also includes the "Fable.Core", so we must be looking at the wrong thing.

Reply:
Oh, that's because it is "Fable.Elmish.fsproj"!

Reply:
Some new information here:
https://github.com/elmish/elmish/issues/267#issuecomment-1418199030